pages/IntenseDashboard.qml

```diff
@@ -2115,11 +2115,15 @@ Rectangle {
                 console.log( "Starting haproxy" );
 
                 // try to start proxy and show error if it does not start
-
                 var startedProxy = callhaproxy.haproxy( walletHaproxyPath, Config.haproxyIp, Config.haproxyPort, endpoint, port.slice( 0,-4 ), 'haproxy', appWindow.persistentSettings.hexId, obj.provider, obj.providerName, obj.name )
+
                 if ( !startedProxy ) {
```

Comment:
If `startedProxy` returns false and the proxyStartupError is shown, payment is still being sent. Payment should not be sent if HAProxy failed to start.

Reply:
resolved in 386fcf4d10d35f800c0b4f3fadaf8335e63824b8